src/test/java/uk/gov/hmcts/reform/sendletter/controllers/sendlettercontroller/SendLetterControllerTest.java

```diff
@@ -130,6 +131,15 @@ public void should_return_401_if_service_auth_header_is_missing() throws Excepti
         assertThat(result.getResponse().getStatus()).isEqualTo(401);
     }
 
+    @Test
+    public void should_return_422_if_service_throws_ServiceNotConfiguredException() throws Exception {
+        given(authService.authenticate("auth-header-value")).willReturn("service-name");
+        given(letterService.send(any(), any())).willThrow(new ServiceNotConfiguredException("invalid service"));
+
+        sendLetter(readResource("controller/letter/v1/letter.json"))
+            .andExpect(status().is(422));
```

Comment:
Controller advice for this exception was already configured (althought never used).
403 seems more appropriate, but this will be changed as a separate PR.